### PR TITLE
feat: adds reminders api

### PR DESCRIPTION
**Pull request by dickie81**
## Description
Provides a simple interface for adding, removing & checking reminders.

Also as a part of this I needed to add test jest framework & fix linting issues.

### Checklist for this pull request
- [x] PR is scoped to one task
- [x] Tests included
- One of:
- [x] The PR is not making any UI change
- [ ] The PR is making a UI change but not a product change
- [ ] Screenshots included

### CODE REVIEW

File: src/reminders/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { NativeModules } from 'react-native';
```
Comment (francois-codes) [inline src/reminders/index.js:1]: DRY !

**Review by francois-codes — CHANGES_REQUESTED** [review]

File: src/reminders/index.js (hunk 1 above)
Comment (dickie81) [reply src/reminders/index.js:1]: Fair - I guess this just becomes a readme change? I was wrapping the methods so things did not crash when run in a context other than and app with the Zapp SDK.

Comment (dickie81) [reply src/reminders/index.js:1]: Is it worth keeping the lint / test framework stuff?

Comment (francois-codes) [reply src/reminders/index.js:1]: wrapping is fine, but you can actually wrap all the methods in one - it's the same syntax an all of them.

something like
```
const methods = ['setReminder', 'removeReminder', ...];
export [...methods].reduce((_export, method) => {
  _export[method] = args => { /* do your stuff */ }
}, {});
```

Comment (dickie81) [reply src/reminders/index.js:1]: tried to DRY it out as much as possible - let me know what you think

**Review by francois-codes — APPROVED** [review]
I like reduce better but that’s your call !

**Review by Budaa — APPROVED** [review]

**Pull request merged**